src/components/Calculator.js

```diff
@@ -1,49 +1,70 @@
     <section className="container">
-      <input disabled value={0} className="show-input" />
+      <input type="text" className="show-input" value={initialValue} readOnly />
       <aside>
         <div>
-          <CalBtn incrementNumber="AC" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="+/-" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="%" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="+" bgColor="bg-orange btn p-normal" />
+          <CalBtn characterStr="AC" bgColor="bg-white btn p-normal" />
+          <CalBtn characterStr="+/-" bgColor="bg-white btn p-normal" />
+          <CalBtn characterStr="%" bgColor="bg-white btn p-normal" />
+          <CalBtn characterStr="+" bgColor="bg-orange btn p-normal" />
         </div>
         <div>
-          <CalBtn incrementNumber="7" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="8" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="9" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="x" bgColor="bg-orange btn p-normal" />
+          <CalBtn incrementNumber={7} bgColor="bg-white btn p-normal" />
+          <CalBtn incrementNumber={8} bgColor="bg-white btn p-normal" />
+          <CalBtn incrementNumber={9} bgColor="bg-white btn p-normal" />
+          <CalBtn characterStr="x" bgColor="bg-orange btn p-normal" />
         </div>
         <div>
-          <CalBtn incrementNumber="4" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="5" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="6" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="-" bgColor="bg-orange btn p-normal" />
+          <CalBtn incrementNumber={4} bgColor="bg-white btn p-normal" />
+          <CalBtn incrementNumber={5} bgColor="bg-white btn p-normal" />
+          <CalBtn incrementNumber={6} bgColor="bg-white btn p-normal" />
+          <CalBtn characterStr="-" bgColor="bg-orange btn p-normal" />
         </div>
         <div>
-          <CalBtn incrementNumber="1" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="2" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="3" bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="+" bgColor="bg-orange btn p-normal" />
+          <CalBtn incrementNumber={1} bgColor="bg-white btn p-normal" />
+          <CalBtn incrementNumber={2} bgColor="bg-white btn p-normal" />
+          <CalBtn incrementNumber={3} bgColor="bg-white btn p-normal" />
+          <CalBtn characterStr="+" bgColor="bg-orange btn p-normal" />
         </div>
         <div>
-          <CalBtn incrementNumber="0" bgColor="bg-white btn p-abnormal" />
-          <CalBtn incrementNumber="." bgColor="bg-white btn p-normal" />
-          <CalBtn incrementNumber="=" bgColor="bg-orange btn p-normal" />
+          <CalBtn incrementNumber={0} bgColor="bg-white btn p-abnormal" />
```

Comment:
- [ ] It would be nice if your `0` should not behave like an equal sign operation :thinking: because right now when you click on it then it adds 0 at the end and suddenly runs the operation of equal :fearful: Kindly fix this issue to make your app work as expected in the requirements :blush: